validphys2/src/validphys/pdfbases.py

```diff
@@ -803,6 +816,18 @@ def fitbasis_to_NN31IC(flav_info, fitbasis):
         g = {'u': 0, 'ubar': 0, 'd': 0, 'dbar': 0, 's': 0, 'sbar': 0, 'c': 0, 'cbar': 0, 'g': 1 }
         v15 = {'u': 1, 'ubar': -1, 'd': 1, 'dbar': -1, 's': 1, 'sbar': -1, 'c': -3, 'cbar': 3, 'g': 0 }
 
+
+    elif fitbasis == 'EVOL_POL':  # With Perturbative Charm
+        sng = {'sng': 1, 'v': 0, 'v3': 0, 'v8': 0, 't3': 0, 't8': 0, 't15': 0, 'g': 0 }
+        v = {'sng': 0, 'v': 1, 'v3': 0, 'v8': 0, 't3': 0, 't8': 0, 't15': 0, 'g': 0 }
```

Comment:
Shouldn't we remove `v,v3,v8` at all for the time being?

Reply:
Same as above, these are needed for the rotation into `NN3IC`. This is perfectly fine since all the non-active flavors will always be zero from the fit anyway.